### ISSUE TITLE
 raise exceptions.from_response(resp, method, url)\nkeystoneauth1.exceptions.http.Unauthorized: The request you have made requires authentication. (HTTP 401)

### ISSUE THREAD

**Issue by ninetwozreo** [switch-ch/jitsi-deploy#2]
` raise exceptions.from_response(resp, method, url)\nkeystoneauth1.exceptions.http.Unauthorized: The request you have made requires authentication. (HTTP 401)`
when I try to exe the command `ansible-playbook build_jibri_server.yml -D`
it come the exceptions, is there any suggests?

**Comment by ninetwozreo** [comment]
as I consider  , I do not know how should I change the `OS_AUTH_URL` in openrc.sample or just not change it

**Comment by jcfischer** [comment]
You will need to use the `openrc` settings from the OpenStack installation you are using and then source it, before you deploy the servers.

**Comment by ninetwozreo** [comment]
so first of all ,I should have a OpenStack Runing.... Sorry, I misunderstood the doc.I did not have it installed ,I just test this on my local VMs .Maybe I should try OpenStack first, and thanks for your attention.

**Comment by jcfischer** [comment]
No!!! OpenStack is the virtualization cloud we are using. If you have running VMs, then use those (and build your VMs the way you normally would). The ansible scripts to build the VMs are very specific to our environment.

**Comment by ninetwozreo** [comment]
I just try to find a easy way to deploy the video-bridge and jibri on separate VMs to support more meet and recording avaliable ,and I found this , can this  ansible scripts help me in that way (sorry for my english)

**Comment by ninetwozreo** [comment]
and  what should I do if I want  skip the problem

**Comment by jcfischer** [comment]
As soon as you have VMs you can use the other ansible scripts (`ansible-playbook -i inventory/production main.yml` to deploy the software. The setup is described here: https://github.com/jitsi/jitsi-meet/blob/master/doc/scalable-installation.md

As said there - you will have to have an understanding of how Jitsi works and a good amount of Linux Sysadmin knowledge

**Comment by ninetwozreo** [comment]
yes, jitsi is a new thing to me ,and thanks for your  guidance ,I will try that

**Comment by ninetwozreo** [comment]
> As soon as you have VMs you can use the other ansible scripts (`ansible-playbook -i inventory/production main.yml` to deploy the software. The setup is described here: https://github.com/jitsi/jitsi-meet/blob/master/doc/scalable-installation.md
> 
> As said there - you will have to have an understanding of how Jitsi works and a good amount of Linux Sysadmin knowledge

As I try to install jibri only , its seems that the `jibri_enabled_servers` did not define, may I ask where should I define this parameter and its format ?
